sessionctx/variable/sysvar.go

```diff
@@ -652,6 +652,15 @@ var defaultSysVars = []*SysVar{
 			return nil
 		},
 	},
+	{Scope: ScopeGlobal, Name: TiDBEnableAutoAnalyze, Value: BoolToOnOff(DefTiDBEnableAutoAnalyze), skipInit: true, Type: TypeBool,
```

Comment:
Skip init is for session scoped vars, and deprecated:
```suggestion
	{Scope: ScopeGlobal, Name: TiDBEnableAutoAnalyze, Value: BoolToOnOff(DefTiDBEnableAutoAnalyze), Type: TypeBool,
```